frontend/vue/components/CodeExercise/CodeEditorTools.vue

```diff
@@ -0,0 +1,107 @@
+<template>
+  <div class="code-editors-tools">
+    <div v-if="resetEnabled" class="code-editors-tools__button">
+      <bx-tooltip-icon
+        class="code-editors-tools__button__tooltip"
+        aligment="center"
+        direction="top"
+        body-text="Reset cell to initial content"
```

Comment:
Similar comment regarding translations, something like:

```
body-text="$translate('Reset cell to initial content')"
```

then make sure the strings are defined in `strings.yaml` file